ci-operator/step-registry/hypershift/conformance/hypershift-conformance-chain.yaml

```diff
@@ -71,11 +71,15 @@ chain:
           ;;
       ovirt) export TEST_PROVIDER='{"type":"ovirt"}';;
       none)
-          if [[ "${DS_IP_STACK}" != "v6" ]];
-          then
-            export TEST_PROVIDER='{"type":"baremetal"}'
+          if [ -f "${SHARED_DIR}/ds-vars.conf" ] ; then
+            if [[ "${DS_IP_STACK}" != "v6" ]];
+            then
+              export TEST_PROVIDER='{"type":"baremetal"}'
+            else
+              mirror_test_images
+              export TEST_PROVIDER='{"type":"baremetal","disconnected":true}'
+            fi
           else
-            mirror_test_images
             export TEST_PROVIDER='{"type":"baremetal","disconnected":true}'
```

Comment:
why this else part is not calling `mirror_test_images`?

Reply:
Moved that else to `if [ -f "${SHARED_DIR}/ds-vars.conf" ]` if block, since `${DS_IP_STACK}` var is defined in this conf file which we don't have and causing error for us. So moved everything inside and kept a standard `TEST_PROVIDER` which was there earlier.